src/inmanta/server/bootloader.py

```diff
@@ -291,11 +317,30 @@ async def wait_for_db(self, db_wait_time: int) -> None:
             "password": config.db_password.get(),
             "database": config.db_name.get(),
         }
+        human_readable_min_pg_version, machine_readable_min_pg_version = self._get_minimal_postgres_version()
+
         while True:
             try:
                 # Attempt to create a database connection
                 conn = await asyncpg.connect(**db_settings, timeout=5)  # raises TimeoutError after 5 seconds
-                LOGGER.info("Successfully connected to the database.")
+                result = await conn.fetch("SHOW server_version_num")
+                pg_server_version_machine_readable = int(result[0]["server_version_num"])
+                result = await conn.fetch("SHOW server_version")
+                pg_server_version_human_readable = result[0]["server_version"]
+
+                if pg_server_version_machine_readable < machine_readable_min_pg_version:
+
+                    LOGGER.warning(
```

Comment:
We need to fail the server startup if the version is too low. (See Slack discussion).